omnigibson/prims/joint_prim.py

```diff
@@ -797,9 +797,14 @@ def set_pos(self, pos, normalized=False, drive=False):
             pos = self._denormalize_pos(pos)
 
         # Set the DOF(s) in this joint
-        if drive:
+        if self.driven and drive:
```

Comment:
Can we add comments here for the semantic differences between these, and the motivations? (same applies to set_vel)

To make this clearer, maybe we should split the if/ else into nested, like so:

```python
if self.driven:
    # Basically any controllable object
    if drive:
        ...
    else:
        ...
else:
    # Any other object, e.g.: furniture
    ...
```